federated-catalog/README.md

```diff
@@ -0,0 +1,60 @@
+# Federated Catalog Samples
+
+The samples in this section focus on the topic of Federated Catalogs.
+
+The Federated Catalog (FC) functions as an aggregated repository of
+catalogs obtained from multiple
+participants in the dataspace. To accomplish this, the FC utilizes crawlers
+that periodically crawl the catalogs from each participant and store this list
+of catalogs in a local cache.
+By maintaining this locally cached version of catalogs, it eliminates the need to query
+each participant individually, resulting in faster and more reliable queries.
+
+
+The following samples shows how to
+* implement, build and run different versions of FC e.g.
+    * standalone,
+    * embedded.
+
+[//]: # (* implement TargetNodeDirectory and resolve Target Nodes,)
+
+[//]: # (    * from a static file containing all participants' DSP endpoints,)
```

Comment:
I'm not an huge fan of comments, they tend to be ignored and stale and become outdated.
Please remove them and just create an issue in this repository reporting what needs to be done.